src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -102,5 +121,11 @@ trait BackendCall {
    * this method will not be called.  If the backend does override `Backend#findResumableExecutions`, the corresponding
    * `BackendCall` should override this method to actually do the resumption work.
    */
-  def resume(jobKey: JobKey): ExecutionResult = ???
+  def resume(jobKey: JobKey)(implicit ec: ExecutionContext): Future[ExecutionHandle] = ???
```

Comment:
If `resume` gets called here something is seriously wrong, as the corresponding `Backend` must have returned a non-empty collection from `findResumableExecutions` without this method being updated to deal with that.  

I agree with changing away from `???`, but I'd lean towards making this a VM-stopping `Error` since this situation really shouldn't be ignored.  

Any other opinions on this?